includes/Core/Storage/Owner_ID.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Storage\Owner_ID
```

Comment:
Let's move this class to `Google\Site_Kit\Core\Authentication\Owner_ID`, similar to where the other authentication-related settings live.

I noticed the other day that the new `Has_Connected_Admins` class was also added to the `Storage` namespace, which was an oversight before, I don't think that should be the case. The `Storage` namespace solely contains base classes, and I think it makes sense to continue to do so.

Reply:
Okay, moved Owner_ID and Has_Connected_Admins to the Authentication namespace.